README.md

```diff
@@ -1458,6 +1458,18 @@ pets.include? 'cat'
   Time.zone.parse('2015-03-02 19:05:37') # => Mon, 02 Mar 2015 19:05:37 EET +02:00
   ```
 
+* <a name="to-time"></a>
+  Don't use [`String#to_time`](https://apidock.com/rails/String/to_time)
+<sup>[[link](#to-time)]</sup>
+
+  ```ruby
+  # bad
+  '2015-03-02 19:05:37'.to_time # => Will also assume time string given is in the system's time zone.
```

Comment:
`=>` construct implies what follows it is returned by the code to the left, and this might be confusing. WDYT of:
```
# bad - assumes time string given is in the system's time zone
'2015-03-02 19:05:37'.to_time

# good - is using time zone from application configuration
  Time.zone.parse('2015-03-02 19:05:37') # => Mon ...
```

I understand that in the same paragraph the confusing syntax is used, probably to be fixed in a follow-up.